locale/res.its

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<its:rules xmlns:its="http://www.w3.org/2005/11/its" version="1.0">
+    <!-- translate textual content of all elements -->
+    <its:translateRule selector="//*" translate="yes"/>
```

Comment:
I understand this simple rule extracts all xml content as strings to translate, but only the <Name> strings really need it, as strings not already part of the program.